internal/cmd/operator-sdk/pkgmantobundle/pkgmantobundle_test.go

```diff
@@ -0,0 +1,239 @@
+// Copyright 2021 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkgmantobundle
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+
+	apimanifests "github.com/operator-framework/api/pkg/manifests"
+	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-sdk/internal/util/projutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Running pkgmanToBundle command", func() {
+	var (
+		p         pkgManToBundleCmd
+		pkgManDir string
+		outputDir string
+	)
+
+	BeforeEach(func() {
+		p = pkgManToBundleCmd{}
+	})
+
+	Describe("validate", func() {
+		It("fail if anything other than one argumanet for packagemanifest directory is provided", func() {
+			err := p.validate([]string{})
+			Expect(err).To(HaveOccurred())
+
+			err = p.validate([]string{"one", "two"})
+			Expect(err).To(HaveOccurred())
+		})
+
+		It("succeeds if exactly one argument is provided", func() {
+			err := p.validate([]string{"inputdir"})
+			Expect(err).NotTo(HaveOccurred())
+		})
+	})
+
+	Describe("migrate packagemanifests to bundle ", func() {
+		AfterEach(func() {
+			err := os.RemoveAll(outputDir)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("should generate multiple bundles for each version of manifests", func() {
+			// Specify input package manifest directory and output directory
+			pkgManDir = filepath.Join("..", "..", "..", "..", "internal", "cmd", "operator-sdk", "pkgmantobundle", "testdata", "packagemanifests")
```

Comment:
This directory is `internal/cmd/operator-sdk/pkgmantobundle`, so shouldn't this just be
```suggestion
			pkgManDir = filepath.Join("testdata", "packagemanifests")
```